### PR TITLE
Moddiconv 13

### PR DESCRIPTION
**Pull request by Igor-Gorchakov**
* Prepared schema and SQL scripts to prepopulate associated profiles in testing mode
* Created DAOs to work with associated profiles
* Testing data representation:
image
* Related PR in data-import-raml-storage: https://github.com/folio-org/data-import-raml-storage/pull/54

### CODE REVIEW

**Comment by Igor-Gorchakov** [comment]
Testing data representation
image

File: mod-data-import-converter-storage-server/src/main/java/org/folio/dao/association/AbstractProfileAssociationDao.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+  public AbstractProfileAssociationDao(ProfileDao masterProfileDao, ProfileDao detailProfileDao) {
+    this.masterProfileDao = masterProfileDao;
+    this.detailProfileDao = detailProfileDao;
+  }
+
+  @Override
+  public Future<String> save(ProfileAssociation entity, String tenantId) {
+    // STUB implementation
+    return Future.succeededFuture();
```
Comment (OleksiiKuzminov) [inline mod-data-import-converter-storage-server/src/main/java/org/folio/dao/association/AbstractProfileAssociationDao.java]: add impls and tests

File: mod-data-import-converter-storage-server/src/test/java/org/folio/rest/impl/ActionProfileTest.java [hunk 2]
```diff
@@ -20,10 +20,7 @@
 import java.util.List;
 import java.util.UUID;
 
-import static org.hamcrest.Matchers.empty;
-import static org.hamcrest.Matchers.everyItem;
-import static org.hamcrest.Matchers.hasItem;
-import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.*;
```
Comment (KaterynaSenchenko) [inline mod-data-import-converter-storage-server/src/test/java/org/folio/rest/impl/ActionProfileTest.java]: Do not use wild card imports

File: mod-data-import-converter-storage-server/src/test/java/org/folio/rest/impl/ActionProfileTest.java [hunk 3]
```diff
@@ -256,9 +253,12 @@ private void createProfiles() {
 
   @Override
   public void clearTables(TestContext context) {
+    Async async = context.async();
     PostgresClient.getInstance(vertx, TENANT_ID).delete(ACTION_PROFILES_TABLE_NAME, new Criterion(), event -> {
       if (event.failed()) {
         context.fail(event.cause());
+      } else {
```
Comment (KaterynaSenchenko) [inline mod-data-import-converter-storage-server/src/test/java/org/folio/rest/impl/ActionProfileTest.java]: looks like 'else' is redundant here

**Pull request merged**